src/elements/content-uploader/ContentUploader.js

```diff
@@ -346,6 +346,58 @@ class ContentUploader extends Component<Props, State> {
         );
     };
 
+    /**
+     * Add dropped items to the upload queue
+     *
+     * @private
+     * @param {DataTransfer} droppedItems
+     * @param {Function} itemUpdateCallback
+     * @returns {Promise<any>}
+     */
+    addDroppedItemsToUploadQueue = (droppedItems: DataTransfer, itemUpdateCallback: Function): void => {
+        if (droppedItems.items) {
+            this.addDataTransferItemsToUploadQueue(droppedItems.items, itemUpdateCallback);
+        } else {
+            this.addDataTransferFilesToUploadQueue(droppedItems.files, itemUpdateCallback);
+        }
+    };
+
+    /**
+     * Add DataTransfer Files to the upload queue
+     *
+     * @private
+     * @param {FileList} dataTransferFiles
+     * @param {Function} itemUpdateCallback
+     * @returns {Promise<any>}
+     */
+    addDataTransferFilesToUploadQueue = (dataTransferFiles: FileList, itemUpdateCallback: Function): void => {
+        if (!dataTransferFiles || dataTransferFiles.length === 0) {
```

Comment:
Is it possible that any of the items are folders?

Reply:
based on my research IE11 can't support drag and drop of folders.  Here's one of the resources I found explaining that the DataTransfer.files object is empty if you drop a folder in IE11.  That can be used to detect a folder was dropped but there's no way to access the actual files in the folder.
[cross browser drag and drop notes](https://github.com/leonadler/drag-and-drop-across-browsers)

Reply:
It seems unfortunate that we need to basically duplicate this logic to account for the fact that the `dataTransferFiles` object is represented in a different form. Can we instead normalize the array to some common file shape and then call to a single `addDataTransferItemsToUploadQueue` method?